backend/src/tasks/test-proxy.ts

```diff
@@ -3,29 +3,34 @@ import { CommandOptions } from './ecs-client';
 import { spawnSync } from 'child_process';
 import { writeFileSync } from 'fs';
 
-const WEBHOOK_URL_HTTP =
-  'http://webhook.site/0f3e7e8f-ffe4-46df-aad1-d017b2c27921';
 const WEBHOOK_URL_HTTPS =
-  'https://webhook.site/0f3e7e8f-ffe4-46df-aad1-d017b2c27921';
+  'https://webhook.site/fef89312-ab36-46fc-bb9b-8a9943f01e55';
```

Comment:
Is this uuid always going to be static? Or could it be an environment variable if it's subject to keep changing?

Reply:
It expires after a bit, so you usually need to regenerate it.

We just run this manually at the moment. Maybe we could make it an env variable in the future, though I think the best long-term solution is what I mentioned in the comment ("TODO: Point these URLs to a locally running web server in order to make this test be able to be automatically run on CI.")

Reply:
Ah ok that makes sense.